assets/RNAseq_report.Rmd

```diff
@@ -31,7 +31,23 @@ params:
 df <- read.csv(file = params$path_proj_summary, sep="\t", header = TRUE)
 # Import the .tsv file downloadable from https://portal.qbic.uni-tuebingen.de/portal/web/qbic/user-database -> Projects
 projectcode = as.character(df$Sub.Project)
+# Set the title; if the title is too long, it would get covered by the logo, so it is split in 2 or 3 parts.
 title = as.character(df$Short.Title)
+#if (nchar(title) > 65 & nchar(title) < 130) {
+  #title1 = paste(substr(title, 1, 65), substr(title, 66, nchar(title)),sep= "\n")
+  #title1 = substr(title, 1, 65)
+  #title2 = substr(title, 66, nchar(title))
+#} else if (nchar(title) > 130) {
+  #title = paste(substr(title, 1, 65), substr(title, 66, 130), substr(title, 131, nchar(title)),sep= "\n")
+  #title1 = substr(title, 1, 65)
+  #title2 = substr(title, 66, 130)
+  #title3 = substr(title, 131, nchar(title))
+#} else {
+#  title1 = title
+#}
+title1 = substr(title, 1, 65)
```

Comment:
Hardcoding the length won't work if the string is shorter than 65

Reply:
I was surprised that this doesn't cause trouble, but apparently it doesn't: `title1` ends up being a string containing the whole subtitle and `title2,3` remain empty. I tried it in R and with the testdata (subtitle ~ 10 characters) no errors are thrown and the report gets generated as normal

Reply:
I will restore the if clause though, it's better, if I can get it to work (the reason why it is set like this now is because I was having troubles with it). It saves also two empty lines after a short subtitles - which don't look that bad to me, but it could be better.